### PR TITLE
Adds Mainnet/Testnet Toggle Component + Render Prop Toggle

### LINKED ISSUE #108: Mainnet/ Testnet Toggle
* [x] **Use RITEway unit tests for TDD**
* [x] **Add Toggle component for Mainet/Testnet**
* [x] **Separate out into a render prop toggle component**
* [x] **Allow for default toggle in render prop**

### LINKED ISSUE #110: Mainnet Toggle Component
* [x] **Fix CSS for the Mainnet Toggle**
* [ ] **Fix failing tests for Travis/Netlify**
* [ ] **Type for rest parameter in callAll function**
* [ ] **Exchange out current Toggle image with new component**

### PR DESCRIPTION
**Pull request by jwicks31**
Resolves #108, Resolves #110 

### Description of Changes

- **Adds ToggleRP render prop component with unit tests**
- **Adds ToggleMainnet component with unit tests**

### PR Review Checklist

* [ ] **Have you tested the changes?**
* [ ] **Is QE review required? If so, has this been reviewed by QE?** (For all new features, review by QE is required. Minor changes which already have passing functional tests may not need a new QE review -- when in doubt, loop in a QE reviewer).
* [ ] **Are there code changes that require new unit tests? If so, are those tests included?** (A separate PR for tests is not appropriate).
* [ ] **Are documentation updates required? If so, does this PR include it?** (Separate documentation PR is generally not appropriate).
* [ ] **Are the added/edited files named appropriately and located correctly in the project directory structure?**

### CODE REVIEW

File: src/components/atoms/ToggleMainnet/ToggleMainnet.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import * as classNames from 'classnames'
+import { ToggleRP } from 'components/atoms/ToggleRP/ToggleRP'
+import { ClassNameProps } from 'interfaces/Props'
+
+interface ToggleMainnetProps extends ClassNameProps {
+  readonly disabled?: boolean
+}
+
+export const ToggleMainnet = (props: ToggleMainnetProps) => {
+  return (
```
Comment (wzalazar) [inline src/components/atoms/ToggleMainnet/ToggleMainnet.tsx]: Can you remove the return? 
is unnecessary, just 
```
=> (
<div>...</div>
)
```

File: src/components/atoms/ToggleRP/ToggleRP.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  border-radius: 45%;
+  background-color: $ghost-white;
+  color: $transparent;
+
+  &:focus {
+    outline: none;
+  }
+}
+
+[role="switch"][aria-checked="true"] :first-child,
```
Comment (wzalazar) [inline src/components/atoms/ToggleRP/ToggleRP.scss]: are you sure is cross browsing?

File: src/components/atoms/ToggleRP/ToggleRP.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  render(): React.ReactNode {
+    const { children } = this.props
+    return children ? (
+      children({
+        on: this.state.on,
+        toggle: this.toggle,
+        getToggleProps: this.getToggleProps,
+      })
+    ) : (
+      <button
```
Comment (wzalazar) [inline src/components/atoms/ToggleRP/ToggleRP.tsx]: sometimes, it's better to use a checkbox for simulating a toggle. I leave the reference link 

https://www.w3schools.com/howto/howto_css_switch.asp

File: src/components/atoms/ToggleRP/ToggleRP.scss (hunk 2 above)
Comment (jwicks31) [reply src/components/atoms/ToggleRP/ToggleRP.scss]: I believe it is. This is what I checked with: https://caniuse.com/#search=%5B

File: src/components/atoms/ToggleRP/ToggleRP.tsx (hunk 3 above)
Comment (ericelliott) [reply src/components/atoms/ToggleRP/ToggleRP.tsx]: We should have a component from a widget library for this. Do we have a component library?

Comment (jwicks31) [reply src/components/atoms/ToggleRP/ToggleRP.tsx]: Is there a certain one we would like to use? And should I use it for this PR?

File: src/components/atoms/ToggleRP/ToggleRP.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  readonly disabled?: boolean
+  readonly on?: boolean
+  readonly onClick?: () => void
+}
+
+interface ToggleState {
+  readonly on?: boolean
+}
+
+const callAll = (...fns: ReadonlyArray<(...args: ReadonlyArray<any>) => void>) => (...args: ReadonlyArray<any>) =>
```
Comment (jwicks31) [inline src/components/atoms/ToggleRP/ToggleRP.tsx]: This is what is breaking the Netlify build. How can we correctly do the Typescript typing for this function? Array is converted to ReadonlyArray to pass the lint rules, but doesn't work with rest parameters.

**Pull request merged**